.github/workflows/docs.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     # disk.
     - name: "Build markdown files from tblgen sources"
       run: |
-        bazel build $(bazel query 'siblings(kind("gentbl_rule", @heir//...))' | grep "_filegroup$")
+        bazel build "$(bazel query 'siblings(kind("gentbl_rule", @heir//...))' | grep "_filegroup$")"
```

Comment:
This looks like it's broken because of the nested double quotes. It also seems to break https://github.com/google/heir/actions/runs/5764646709/job/15629048762?pr=87

Reply:
Ah, I see. I just rewrote the grep to be a bazel query filter and piped those into `bazel build`. shellcheck wasn't happy with the original being unquoted.